src/o365/spo/commands/orgassetslibrary/orgassetslibrary-add.ts

```diff
@@ -0,0 +1,140 @@
+        };
+
+        return request.post(requestOptions);
+      })
+      .then((res: string): void => {
+        const json: ClientSvcResponse = JSON.parse(res);
+          const response: ClientSvcResponseContents = json[0];
+          if (response.ErrorInfo) {
+            cb(new CommandError(response.ErrorInfo.ErrorMessage));
+            return;
+          }
+          else {
+            cmd.log(json[json.length - 1]);
+
+            if (this.verbose) {
+              cmd.log(vorpal.chalk.green('DONE'));
+            }
+          }
+          cb();
+      }, (err: any): void => this.handleRejectedPromise(err, cmd, cb));
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (!args.options.libraryUrl) {
+        return `Required parameter libraryUrl missing`;
+      }
+
+      if (typeof args.options.thumbnailUrl !== 'undefined' && SpoCommand.isValidSharePointUrl(args.options.thumbnailUrl) != true) {
+        return SpoCommand.isValidSharePointUrl(args.options.thumbnailUrl);
+      }
+
+      return SpoCommand.isValidSharePointUrl(args.options.libraryUrl);
+
+    };
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '--libraryUrl <libraryUrl>',
+        description: 'The URL of the library to promote'
+      },
+      {
+        option: '--thumbnailUrl [thumbnailUrl]',
+        description: 'The URL of the thumbnail to render'
+      },
+      {
+        option: '--cdnType [cdnType]',
+        description: 'Specifies the CDN type. Public|Private. Default Private',
+        autocomplete: ['Public', 'Private']
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public commandHelp(args: CommandArgs, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(commands.ORGASSETSLIBRARY_ADD).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} to use this command you have to have permissions to access
+    the tenant admin site.
+
+  Examples:
+
+  Promotes an existing library to become an organization assets library
+    ${commands.ORGASSETSLIBRARY_ADD} -libraryUrl https://contoso.sharepoint.com/assets
+
+  Promotes an existing library to become an organization assets library with Thumbnail
+    ${commands.ORGASSETSLIBRARY_ADD} -libraryUrl https://contoso.sharepoint.com/assets -thumbnailUrl https://contoso.sharepoint.com/assets/logo.png
```

Comment:
Could you please change `-libraryUrl` to `--libraryUrl`  and also `-thumbnailUrl` to `--thumbnailUrl` ?